From 79ff81df7b4593769ec40539e7a669e7d038efd1 Mon Sep 17 00:00:00 2001
From: Mike Hommey <mh@glandium.org>
Date: Fri, 5 Apr 2024 05:59:15 +0900
Subject: [PATCH 2/4] Revert "[InstallAPI][test] Add requires x86_64 for
 hardcoded target test"

This reverts commit bdb60e6f0c8e89abf9bdf36411348db304ca65ba.
---
 clang/test/InstallAPI/diagnostics-dsym.test | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clang/test/InstallAPI/diagnostics-dsym.test b/clang/test/InstallAPI/diagnostics-dsym.test
index 8a1b394f2f86..45c69c09d286 100644
--- a/clang/test/InstallAPI/diagnostics-dsym.test
+++ b/clang/test/InstallAPI/diagnostics-dsym.test
@@ -1,4 +1,4 @@
-; REQUIRES: 86_64-darwin
+; REQUIRES: system-darwin
 
 ; RUN: rm -rf %t
 ; RUN: split-file %s %t
-- 
2.44.0.1.g9765aa7075

